cargo-concordium/src/main.rs

```diff
@@ -442,10 +471,25 @@ pub fn main() -> anyhow::Result<()> {
                 utils::WasmVersion::V1 => handle_run_v1(*run_cmd, module)?,
             }
         }
-        Command::Test { args, seed } => {
-            let success =
-                build_and_run_wasm_test(&args, seed).context("Could not build and run tests.")?;
-            ensure!(success, "Test failed");
+        Command::Test {
+            seed,
+            build_options,
+            only_unit_tests,
+        } => {
+            // TODO: Fail early if wasm version == 0?
```

Comment:
I think we don't need to do that. Maybe they have written some tests that work with V0 modules in some other way.

Does that make sense?

Reply:
Yes, that is reasonable. I'll remove the TODO.